.github/actions/notify-build-status/README.md

```diff
@@ -79,9 +79,9 @@ jobs:
       - id: check
         uses: elastic/apm-pipeline-library/.github/actions/check-dependent-jobs@current
         with: ${{ toJSON(needs) }}
-      - run: ${{ steps.check.outputs.isSuccess }}
       - uses: elastic/apm-pipeline-library/.github/actions/notify-build-status@current
         with:
+          status: "${{ steps.check.outputs.isSuccess ? 'success' : 'failure' }}"
```

Comment:
ah, I forgot that github actions doesn't have ternary logic. WIP

Reply:
https://github.com/elastic/apm-pipeline-library/pull/2130/commits/4dc99ab0707c2ce6b2fd3719b2fc411ee33a7db3